mackerel-plugin-fluentd/lib/fluentd.go

```diff
@@ -298,6 +335,7 @@ func Do() {
 	prefix := flag.String("metric-key-prefix", "fluentd", "Metric key prefix")
 	tempFile := flag.String("tempfile", "", "Temp file name")
 	extendedMetricNames := flag.String("extended_metrics", "", "extended metric names joind with ',' or 'all' (fluentd >= v1.6.0)")
+	workers := flag.Uint("workers", 1, "specifying the number of Fluentd's multi-process workers")
```

Comment:
[ask] Do you think there is a possibility that users of the plugin would like to use the metric name like `custom.fluentd.worker1.#.*` even if `-workers=1`?

If so, I think it is better to set default value to `0`. If not, I think `1` is good value.

Reply:
Thanks for your review.

I think it's unlikely, because that users with workers=1 will not be aware of workers.
The default number of workers in fluentd is one.

ref. https://docs.fluentd.org/deployment/multi-process-workers#multi-process-workers-and-plugins

> By default, one instance of `fluentd` launches a supervisor and a worker. A worker consists of input/filter/output plugins.

Also, if users want to get the metrics of a specific worker, they can use the `-metric-key-prefix` option and the `-port` option.

Reply:
I see, it looks good.